aws/cloudformation/iam.yml.erb

```
@@ -211,11 +213,57 @@ Resources:
       Tags:
         - Key: sqlworkbench-team
           Value: codedotorg
-  DataAnalystRole:
+
+# Console & CLI Roles
+
+<%
+roles = {
+  EngineeringReadOnlyRole: {
+    RoleName: "Engineering_ReadOnly",
+    GoogleIdsSSMParameter: 'DeveloperGoogleIDs'
```

nit: a few missing trailing commas
```suggestion
    GoogleIdsSSMParameter: 'DeveloperGoogleIDs',
```